site/docs/v1/tech/tutorials/integrate-dotnet-api.md

```diff
@@ -0,0 +1,191 @@
+```
+
+Then, create a {{page.technology}} project:
+
+```shell
+dotnet new console --output SetupFusionauth && cd SetupFusionauth
+```
+
+Next, you’ll need to import a few NuGet packages:
+
+```shell
+dotnet add package JSON.Net
+dotnet add package FusionAuth.Client 
+```
+
+Copy and paste the following code into `Program.cs`.
+
+```csharp
+{% remote_include https://raw.githubusercontent.com/FusionAuth/fusionauth-example-client-libraries/main/dotnet/Program.cs %}
+```
+
+Update the `application.oauthConfiguration.authorizedRedirectURLs` value to `http://localhost:3000/profile.html`. Update the `application.oauthConfiguration.logoutURL` value to `http://localhost:3000`.
+
+Now you can publish and run the application. Replace `<YOUR_API_KEY>` with the API key that you generated.
+
+{% include _callout-note.liquid content="The setup script is designed to run on a newly installed FusionAuth instance with only one user and no tenants other than `Default`. To follow this guide on a FusionAuth instance that does not meet these criteria, you may need to modify the above script. <br><br> Refer to the  [.NET Core client library](/docs/v1/tech/client-libraries/netcore) documentation for more information.<br><br>The path to the SetupFusionauth executable will be different depending on your platform." %}
+
+```shell
+dotnet publish -r osx-x64
+fusionauth_api_key=<YOUR_API_KEY> bin/Debug/net7.0/osx-x64/publish/SetupFusionauth 
+```
+
+If you are using PowerShell, you will need to set the environment variable in a separate command before executing the script.
+
+```shell
+$env:fusionauth_api_key='<your API key>'
+```
+
+If you want to, you can [log in to your instance](http://localhost:9011) and examine the new API configuration the script created for you. Navigate to the <span class="breadcrumb">Applications</span> tab to do so.
+
+## Create Your {{page.technology}} API
+
+Now you are going to create a {{page.technology}} API. While this section builds a simple {{page.technology}} API, you can use the same configuration to build a more complex {{page.technology}} API.
+
+First, create the skeleton of the {{page.technology}} API. The .NET framework has a generator to build this out.
+
+```shell
+dotnet new webapi -o MyAPI && cd MyAPI
+```
+
+Then, you’ll need to import a few NuGet packages:
+
+```shell
+dotnet add package Microsoft.AspNetCore.Authentication.JwtBearer
+dotnet add package System.IdentityModel.Tokens.Jwt
+```
+
+You can now start writing the code for your {{page.technology}} API. First, let's create a controller that gives back a JSON message. Create a file called `IdentityController.cs` in the `Controllers` folder, and add the following code to it.
+
+```csharp
+{% remote_include https://raw.githubusercontent.com/ritza-co/fusionauth-dotnet-api-example/main/MyAPI/Controllers/IdentityController.cs %}
+```
+
+This controller returns a JSON object with the identity and claims of the user accessing the service.
```

Comment:
Please change this to be the messages json as documented here: https://fusionauth.io/docs/v1/tech/tutorials/integrate-ruby-rails-api

I think it is a bit confusing to be iterating the claims of a JWT as part of the API. Let's keep the API content simple.